systemvm/debian/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -453,8 +453,8 @@ def fw_router(self):
                 ["", "", "-A NETWORK_STATS_%s -o %s ! -i eth0 -p tcp" % (self.dev, self.dev)])
             self.fw.append(
                 ["", "", "-A NETWORK_STATS_%s -i %s ! -o eth0 -p tcp" % (self.dev, self.dev)])
-            self.fw.append(["nat", "",
-                            "-A POSTROUTING -o %s -j SNAT --to-source %s" % (self.dev, self.cl.get_eth2_ip())])
+            self.fw.append(
+                ["nat", "", "-A POSTROUTING -o %s -j SNAT --to-source %s" % (self.dev, self.cl.get_eth2_ip())])
```

Comment:
is this an intended change?

Reply:
Just a reformat, nothing changed actually.
Can be reverted

Reply:
ok, never mind.